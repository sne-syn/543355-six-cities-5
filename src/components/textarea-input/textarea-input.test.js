import React from 'react';
import renderer from 'react-test-renderer';
import TextareaInput from './textarea-input';

const noop = () => {};
test(`Render TextareaInput`, () => {
  const tree = renderer
    .create(<TextareaInput handleInputChange={noop}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
