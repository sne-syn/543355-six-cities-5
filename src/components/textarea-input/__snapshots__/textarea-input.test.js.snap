// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render TextareaInput 1`] = `
<textarea
  className="reviews__textarea form__textarea"
  id="review"
  name="review"
  onChange={[Function]}
  placeholder="Tell how was your stay, what you like and what can be improved"
/>
`;
